### PR TITLE
optimize: optimize build and fix the base image

### LINKED ISSUE #4960: 不支持ARM64
seataio/seata-server:latest版本docker镜像，不支持树莓派ARM64位
image

### PR DESCRIPTION
**Pull request by wangliang181230**
- [x] I have registered the PR [changes](https://github.com/seata/seata/tree/develop/changes).

### Ⅰ. Describe what this PR did
optimize: optimize build and fix the base image.
优化：优化构建配置，并修正docker镜像的基础镜像.

### Ⅱ. Does this pull request fix one issue?

fixes #4960

### CODE REVIEW

File: build/pom.xml [hunk 1]
```diff
@@ -234,6 +234,7 @@
                         <exclude>**/META-INF/additional-spring-configuration-metadata.json</exclude>
                         <exclude>protobuf/**</exclude>
                         <exclude>**/*.proto</exclude>
+                        <exclude>static/console-fe/**</exclude>
```
Comment (wangliang181230) [inline build/pom.xml:237]: 前端源码也被打包进 `seata-console.jar` 中了，排除掉。

**Review by a364176773 — APPROVED** [review]
LGTM

File: .github/workflows/publishes.yml [hunk 2]
```diff
@@ -46,11 +46,11 @@ jobs:
       - name: "Package and Publish-Image-To-DockerHub"
         if: matrix.type == 'image'
         env:
+          IMAGE_NAME: openjdk:8u342
           REGISTRY_USERNAME: ${{ secrets.REGISTRY_USERNAME }}
           REGISTRY_PASSWORD: ${{ secrets.REGISTRY_PASSWORD }}
         run: |
-          echo "IMAGE_NAME=openjdk:8u332" >> $GITHUB_ENV;
-          if [ "${{github.ref}}" == "refs/heads/snapshot" ]; then
+          if [ "${{ github.ref_name }}" == "develop" ] || [ "${{ github.ref_name }}" == "snapshot" ]; then
```
Comment (slievrly) [inline .github/workflows/publishes.yml:53]: why need it ?`"${{ github.ref_name }}" == "develop" `

**Review by slievrly — APPROVED** [review]
LGTM

File: .github/workflows/publishes.yml (hunk 2 above)
Comment (wangliang181230) [reply .github/workflows/publishes.yml:53]: This schedule only takes effect in the default branch `develop`.

**Pull request merged**